pkgs/applications/networking/cluster/werf/default.nix

```diff
@@ -52,9 +59,15 @@ buildGoModule rec {
   '';
 
   meta = with lib; {
-    homepage = "https://github.com/werf/werf";
     description = "GitOps delivery tool";
+    longDescription = ''
+      The CLI tool gluing Git, Docker, Helm & Kubernetes with any CI system to
+      implement CI/CD and Giterminism.
+    '';
+    homepage = "https://github.com/werf/werf";
+    changelog = "${meta.homepage}/releases/tag/${src.rev}";
```

Comment:
Please don't substitute homepage here so that people can easily clock on the link and it does not get broken by accident.